src/compiler/types.ts

```diff
@@ -4608,26 +4608,24 @@ export type FileIncludeReason =
 
 /** @internal */
 export const enum FilePreprocessingDiagnosticsKind {
-    FilePreprocessingReferencedDiagnostic,
+    FilePreprocessingLibreferenceDiagnostic,

```

Comment:
The casing of `Libreference` seems inconsistent between other places in the code where we say `LibReference`.